opentelemetry-sdk/Cargo.toml

```diff
@@ -30,6 +30,7 @@ url = { version = "2.2", optional = true }
 tokio = { version = "1.0", default-features = false, features = ["rt", "time"], optional = true }
 tokio-stream = { version = "0.1", optional = true }
 http = { version = "0.2", optional = true }
+lazy_static = "1.4"
```

Comment:
nit: can we use [once_cell](https://docs.rs/once_cell/latest/once_cell/) over lazy_static? more of the standard now that it is being added to std.